### DISCUSSION TITLE
[pt] token.conjuncts not working in 3.3 for Portuguese

### DISCUSSION THREAD

**Discussion by ricardojosehlima** [explosion/spaCy#10811] [Help: Other Questions]
when in Spacy 3.2.2
frase = "O menino e a menina saíram." # "The boy and the girl left."
nlp = spacy.load("pt_core_news_lg")
doc = nlp(frase)

comp = doc[1].conjuncts
print([t.text for t in comp])
['menina']

now, in 3.3
same code, result
[]

**Comment by ricardojosehlima** [comment]
Oh this may be due to now "menina" not having conj as dep_ but nsubj, along with "menino"!?

**Comment by adrianeboyd** [comment]
There are can be differences in the model outputs for different model versions (here: `pt_core_news_lg` v3.2.0 vs. v3.3.0), so I don't think this is a bug in `token.conjuncts` (which hasn't changed in spacy itself) but instead an error/difference in the parser output.
